src/views/admin/KlasseCreationView.vue

```diff
@@ -26,127 +20,40 @@
     type Router,
   } from 'vue-router';
 
-  const { t }: Composer = useI18n({ useScope: 'global' });
   const router: Router = useRouter();
-  const authStore: AuthStore = useAuthStore();
-  const personStore: PersonStore = usePersonStore();
   const organisationStore: OrganisationStore = useOrganisationStore();
-  // get personenuebersicht for the current user, if we don't have it
-  watch(
-    () => ({
-      uebersicht: personStore.personenuebersicht,
-      currentUser: authStore.currentUser,
-    }),
-    async ({
-      uebersicht,
-      currentUser,
-    }: {
-      uebersicht: PersonStore['personenuebersicht'];
-      currentUser: AuthStore['currentUser'];
-    }) => {
-      if (!(currentUser && currentUser.personId)) return;
-      if (uebersicht && uebersicht.personId === currentUser.personId) return;
-
-      await personStore.getPersonenuebersichtById(currentUser.personId);
-    },
-    { immediate: true },
-  );
+  // eslint-disable-next-line @typescript-eslint/typedef
+  const formRef = useTemplateRef('klasse-creation-form');
```

Comment:
```suggestion
  const formRef: ShallowRef<InstanceType<typeof KlasseForm> | null> = useTemplateRef('klasse-creation-form');
```

useTemplateRef erstellt ein ShallowRef aus der Form vom Typ KlasseForm. Das hier oben sollte funktionieren.